Automattic/jetpack
# CRM:  Change edit pages to fit Jetpack Emerald style

## Proposed changes:
* This PR changes the following pages to fit Jetpack Emerald style:
    * Edit Contact
    * Edit Company
    * Edit Quote
    * Edit Quote Template
    * Edit Transactions 

### Other information:


## Jetpack product discussion
[Change CRM edit pages to fit the Emerald style#3124](https://github.com/Automattic/zero-bs-crm/issues/3124)

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
* Go to the following pages and make sure it fits the latest Jetpack Emerald style:
 _(Note: test it while editing existing content, and also adding new content)_
    * Edit Contact
    * Edit Company
    * Edit Quote
    * Edit Quote Template
    * Edit Transactions 

## Old vs New screenshots:

![image](https://github.com/Automattic/jetpack/assets/37049295/1503bf61-781c-4aa4-8c25-c7d7b2b2b14b)
![image](https://github.com/Automattic/jetpack/assets/37049295/9af9acdf-1dc3-4cf6-a59c-a70201db6e89)

![image](https://github.com/Automattic/jetpack/assets/37049295/3b90ec6b-cf60-4179-b8a8-ac92bcc47d47)
![image](https://github.com/Automattic/jetpack/assets/37049295/f1718d2c-eb0e-4a6b-85b7-d1f64158a9b8)

![image](https://github.com/Automattic/jetpack/assets/37049295/d3d75e01-aa67-4c03-8fbf-a2f4e6b6b233)
![image](https://github.com/Automattic/jetpack/assets/37049295/3f9ac4b5-ec40-4d8f-99c6-06279d723b03)

![image](https://github.com/Automattic/jetpack/assets/37049295/da25ac76-e37e-431f-8672-f8bc2676d898)
![image](https://github.com/Automattic/jetpack/assets/37049295/ba6add7b-3c10-4fcd-ae5a-24d2750be1c0)

![image](https://github.com/Automattic/jetpack/assets/37049295/abcda520-32cf-4068-b88a-b6e4e8364d56)
![image](https://github.com/Automattic/jetpack/assets/37049295/7de53db3-4d47-4b58-bebd-ebd03732fd5d)
